_i18n/fr.yml

```diff
@@ -49,6 +49,7 @@ help_subpages:
   privacy-and-security: Confidentialité et sécurité
   signing-in: Connexion
   trusted-traveler-programs: Trusted Traveler Programs(Needs translation)
+  sam: System for Award Management (Needs translation)
```

Comment:
If something needs translation, don't we just leave it as NEEDS TRANSLATION and not fill in any text?

Reply:
Oh, wow, I see this pattern all over the trusted traveler content. I'm pretty sure that's now how our internationalization is set up.

Reply:
I'm gonna remove these menu items in a fresh commit and confirm that the text remains untranslated before completing the review.